### PR TITLE
add ChannelsManager#{sendMessage, editMessage} and Command#onError

### PR DESCRIPTION
**Pull request by DjDeveloperr**
Currently, its not possible to send message to a channel which is not cached. This is a limitation and needs to be removed. Only workaround was using `client.rest`. Now, I've added `ChannelsManager#sendMessage` and `editMessage` which can simply take channel ID too. `TextChannel#send` now depends on this new method.

### CODE REVIEW

File: src/managers/channels.ts [hunk 1]
```diff
@@ -66,4 +76,93 @@ export class ChannelsManager extends BaseManager<ChannelPayload, Channel> {
         .catch((e) => reject(e))
     })
   }
+
+  async sendMessage(
+    channel: string | TextChannel,
+    content?: string | AllMessageOptions,
+    option?: AllMessageOptions,
+    reply?: Message
```
Comment (Helloyunho) [inline src/managers/channels.ts:84]: I think adding reply to message option will be better

File: src/managers/channels.ts [hunk 2]
```diff
@@ -66,4 +76,93 @@ export class ChannelsManager extends BaseManager<ChannelPayload, Channel> {
...
+        : channel
+    const res = new Message(this.client, resp, chan, this.client.user as any)
+    await res.mentions.fromPayload(resp)
+    return res
+  }
+
+  async editMessage(
+    channel: string | TextChannel,
+    message: Message | string,
+    text?: string,
```
Comment (Helloyunho) [inline src/managers/channels.ts:135]: make users to allow to use message options via text arg

**Review by Helloyunho — CHANGES_REQUESTED** [review]

File: src/managers/channels.ts (hunk 1 above)
Comment (DjDeveloperr) [reply src/managers/channels.ts]: done

File: src/managers/channels.ts (hunk 2 above)
Comment (DjDeveloperr) [reply src/managers/channels.ts]: done

**Review by Helloyunho — APPROVED** [review]
lgtm!

**Comment by DjDeveloperr** [comment]
Added `BaseManager#resolve`, can you review that

**Review by Helloyunho — APPROVED** [review]

**Pull request merged**